spec.html

```diff
@@ -29918,68 +29930,68 @@ <h1>Runtime Semantics: GetSubstitution ( _matched_, _str_, _position_, _captures
+                  `$0`
                 </td>
               </tr>
               <tr>
                 <td>
-                  0x0024, N, N
-                  <br>
-                  Where
+                  0x0024, N
                   <br>
-                  0x0030 &le; N &le; 0x0039
+                  where 0x0031 &le; N &le; 0x0039
                 </td>
                 <td>
-                  `$nn` where
-                  <br>
-                  `n` is one of `0 1 2 3 4 5 6 7 8 9`
+                  `$`_n_
+                  where _n_ is an expansion of |NonZeroDigit|
                 </td>
                 <td>
-                  The _nn_<sup>th</sup> element of _captures_, where _nn_ is a two-digit decimal number in the range 01 to 99. If _nn_ &le; _m_ and the _nn_<sup>th</sup> element of _captures_ is *undefined*, use the empty String instead. If _nn_ is 00 or _nn_ &gt; _m_, no replacement is done.
+                  If the MV of _n_ &gt; _m_, the replacement is the matched code units (equivalent to no replacement).
+                  <br>
+                  Otherwise, if the element in _captures_ at index equal to the MV of _n_ minus 1 is *undefined*, the replacement is the empty String.
+                  <br>
+                  Otherwise, the replacement is the element in _captures_ at index equal to the MV of _n_ minus 1.
                 </td>
               </tr>
               <tr>
                 <td>
-                  0x0024, 0x003C
+                  0x0024, N1, N2
+                  <br>
+                  where 0x0030 &le; N1 &le; 0x0039 and 0x0030 &le; N2 &le; 0x0039
                 </td>
                 <td>
-                  `$&lt;`
+                  `$`_nn_
+                  where _nn_ is a two-element expansion of |DecimalDigits|
                 </td>
                 <td>
-                  <emu-alg>
-                    1. If _namedCaptures_ is *undefined*, the replacement text is the String `"$&lt;"`.
-                    1. Else,
-                      1. Scan until the next `>` U+003E (GREATER-THAN SIGN).
-                      1. If none is found, the replacement text is the String `"$&lt;"`.
-                      1. Else,
-                        1. Let _groupName_ be the enclosed substring.
-                        1. Let _capture_ be ? Get(_namedCaptures_, _groupName_).
-                        1. If _capture_ is *undefined*, replace the text through `>` with the empty string.
-                        1. Otherwise, replace the text through `>` with ? ToString(_capture_).
-                  </emu-alg>
+                  If _nn_ is `00` or the MV of _nn_ &gt; _m_, the replacement is the string-concatenation of the replacement for the first two matched code units (i.e., a replacement specified by one of the two preceding rows) and the remaining matched code units (i.e., a single |DecimalDigit|).
```

Comment:
I want to leave this, because the intent is to split the matched code units into two subsequences, replacing the first and preserving the second (which _happens_ to be a single code unit, but is not required to be so for the logic to work).